code/modules/mob/living/simple_animal/bot/floorbot.dm

```diff
@@ -250,10 +257,10 @@
 
 /mob/living/simple_animal/bot/floorbot/proc/is_hull_breach(turf/t) //Ignore space tiles not considered part of a structure, also ignores shuttle docking areas.
 	var/area/t_area = get_area(t)
-	if(t_area && (t_area.name == "Space" || findtext(t_area.name, "huttle")))
-		return 0
+	if(t_area && (t_area.name == "Space"))
+		return FALSE
 	else
-		return 1
+		return TRUE
```

Comment:
```
return !istype(get_area(t), /area/space)

Reply:
bc3a71a
This is what you mean?

Reply:
Nope, replace the whole thing

Reply:
Ah alright. wasnt sure, ty

Reply:
[0fa7acf](https://github.com/ParadiseSS13/Paradise/pull/23162/commits/0fa7acf835291a824387d9601f76be3a51f35064)